racket/collects/net/http-client.rkt

```diff
@@ -97,10 +103,15 @@
   ;; (thread (λ () (copy-port log-i to (current-error-port))))
 
   (set-http-conn-to! hc to)
-  (set-http-conn-from! hc from))
+  (set-http-conn-from! hc from)
+
+  (set-http-conn-auto-reconnect?! hc auto-reconnect?)
+  (set-http-conn-auto-reconnect-host! hc host-bs)
```

Comment:
I think we can actually just use the `host` field and don't need `auto-reconnect-host`. Is there something I'm missing?